dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -206,7 +211,9 @@ private static ArtifactResults generateArtifactReport(Configuration configuratio
       templateData.put("upperBoundFailures", upperBoundFailures);
       templateData.put("globalUpperBoundFailures", globalUpperBoundFailures);
       templateData.put("lastUpdated", LocalDateTime.now());
-      templateData.put("dependencyTree", dependencyTree);
+      // Explicit casting avoids Freemarker's error on `AbstractListMultimap.get` in CircleCI
```

Comment:
Can you explain this a bit more? How does Freemarker know anything about this cast?